app/assets/javascripts/questions.js

```diff
@@ -10,4 +10,16 @@ $(document).ready(function(q){
       $('.other-question').removeClass('hidden');
     });
 
+// Runs when the page loads and disables the submit button. Checks that both the answer and question text boxes are blank
+    if($('#question_text').val() || $('#question_answers_attributes_0_text').val() ==  "") 
+   $('#submit_question').attr('disabled', true);
+
+// Every time a key is pressed, this code runs and checks the contents of the answer and question text boxes. If both are full then the submit button is enabled 
+  $('#question_text, #question_answers_attributes_0_text').keyup(function(){
+    if($('#question_text').val() && $('#question_answers_attributes_0_text').val() !=  "") 
```

Comment:
Similar to the above 4-line if statement, this can be condensed, but for readability's sake it's probably better to use two lines:
```
var answerTextVal = $('#question_text').val() && $('#question_answers_attributes_0_text').val();
$('#submit_question').attr('disabled', !answerTextVal);
```
You could do something similar to the other 4-line if block (use two lines) if you feel it's more readable.
If you stick with 4 lines, use curly braces.